Common/Utils/src/ConfigurableParamHelper.cxx

```diff
@@ -143,6 +143,16 @@ const char* asString(TDataMember const& dm, char* pointer)
   // first check if this is a basic data type, in which case
   // we let ROOT do the work
   if (auto dt = dm.GetDataType()) {
+    // we put the numeric interpration for char / unsigned char
+    // instead of the string one
+    if (dt->GetType() == EDataType::kChar_t) {
+      auto c = (char)(*pointer);
+      return std::to_string((int)c).c_str(); // safe?? or do we need to alloc string on heap?
```

Comment:
This will not work, as the mac CI tells. The string is created on the stack, and then a pointer to it is returned, which is invalid since the string is a temporary and immediately removed. Either we'll have to return the string as std::string object, or some case of static variable trick could be used, but that would not be thread save.